src/CSharp/MetadataWebApi/MetadataWebApi/MetadataWebApi.csproj

```diff
@@ -1,118 +1,23 @@
+    <Copyright>Experian 2017</Copyright>
+    <ApplicationIcon>Experian.ico</ApplicationIcon>
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Configuration" />
-    <Reference Include="System.Net.Http">
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Net.Http.Formatting, Version=5.2.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\Microsoft.AspNet.WebApi.Client.5.2.3\lib\net45\System.Net.Http.Formatting.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Net.Http.WebRequest">
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Runtime.Serialization">
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="..\CommonAssemblyInfo.cs">
-      <Link>CommonAssemblyInfo.cs</Link>
-    </Compile>
-    <Compile Include="DataFile.cs" />
-    <Compile Include="FileDownloadReply.cs" />
-    <Compile Include="GetDownloadUriRequest.cs" />
-    <Compile Include="GlobalSuppressions.cs" />
-    <Compile Include="IFileStore.cs" />
-    <Compile Include="IMetadataApi.cs" />
-    <Compile Include="IMetadataApiExtensions.cs" />
-    <Compile Include="IMetadataApiFactory.cs" />
-    <Compile Include="LocalFileStore.cs" />
-    <Compile Include="MetadataApi.cs" />
-    <Compile Include="MetadataApiException.cs" />
-    <Compile Include="MetadataApiFactory.cs" />
-    <Compile Include="Package.cs" />
-    <Compile Include="PackageGroup.cs" />
-    <Compile Include="Program.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="..\MetadataWebApi.snk">
-      <Link>MetadataWebApi.snk</Link>
-    </None>
-    <None Include="..\MetadataWebApi.targets">
-      <Link>MetadataWebApi.targets</Link>
-    </None>
-    <None Include="app.config">
-      <SubType>Designer</SubType>
-    </None>
-    <None Include="app.manifest" />
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <CodeAnalysisDictionary Include="..\FxCopDictionary.xml">
-      <Link>FxCopDictionary.xml</Link>
-    </CodeAnalysisDictionary>
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="1.1.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="system.runtime.serialization.xml" Version="4.3.0" />
+    <PackageReference Include="system.xml.xmlserializer" Version="4.3.*" />
```

Comment:
Related to above, it's a VS2017 format but will work with Code or dotnet cmd as well.